dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/OSUtils.java

```diff
@@ -68,6 +68,15 @@ public static boolean isWindows() {
         return getOSName().startsWith("Windows");
     }
 
+    /**
+     * whether is linux
+     *
+     * @return true if linux
+     */
+    public static boolean isLinux() {
+        return !isWindows() && !isMacOS();
+    }
```

Comment:
It’s better to directly use `SystemUtils.IS_OS_LINUX`, this may need to add `commons.lang3` in task api module.

Reply:
Done.